CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/EddieHubCommunity/) [![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/EddieHubCommunity/)
+
+- Take a look at the existing [Issues](https://github.com/EddieHubCommunity/EventCalendar/issues) or [create a new issue](https://github.com/EddieHubCommunity/EventCalendar/issues/new/choose)!
+- [Fork the Repo](https://github.com/EddieHubCommunity/EventCalendar/fork), create a branch for any issue that you are working on and commit your work.
+- Create a **[Pull Request](https://github.com/EddieHubCommunity/EventCalendar/compare)** (*PR*), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes that are included in your commits.
+
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a fork the [**EventCalendar**](https://github.com/EddieHubCommunity/EventCalendar) repository. Click on the <a href="https://github.com/EddieHubCommunity/EventCalendar/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a> symbol at the top right corner.
+
+**2.** Clone your new fork of the repository:
+
+```bash
+git clone https://github.com/<your-github-username>/EventCalendar
+```
+
+**3.** Set upstream command:
+```bash
+git remote add upstream https://github.com/EddieHubCommunity/EventCalendar.git
+```
+
+**4.** Navigate to the new project directory:
+
+```bash
+cd EventCalendar
+```
+
+**5.** Create a new branch:
+```bash
+git checkout -b YourBranchName
+```
+
+**6.** Sync your fork or local repository with the origin repository:
+- In your forked repository click on "Fetch upstream"
+- Click "Fetch and merge".
+
+
+### Alternatively, Git CLI way to Sync forked repository with origin repository:
+```bash
+git fetch upstream
+```
+```bash
+git merge upstream/main
+```
+### [Github Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for Syncing
```

Comment:
Why is this a heading?

Reply:
I followed the format of [LinkFree](https://github.com/EddieHubCommunity/LinkFree), should I change it to normal text?

Reply:
Ah I see, if it were added there as a heading it must be for some reason 🤔